airbyte-integrations/connectors/source-firebase-realtime-database/integration_tests/catalog.json

```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
+      "supported_sync_modes": ["full_refresh", "incremental"],
+      "source_defined_cursor": true,
+      "default_cursor_field": "column1",
```

Comment:
Can you fill in the values in this file?

Reply:
fixed in https://github.com/airbytehq/airbyte/pull/18029/commits/ae71001b07290a3d91b24d2fc195e5b096d76020